File: src/main/kotlin/com/sourcegraph/cody/agent/protocol_generated/ChatExportResult.kt
@file:Suppress("FunctionName", "ClassName", "unused", "EnumEntryName", "UnusedImport")
package com.sourcegraph.cody.agent.protocol_generated;

data class ChatExportResult(
  val chatID: String,
  val transcript: SerializedChatTranscript,
)

File: agent/bindings/csharp/ChatExportResult.cs
using System.Text.Json.Serialization;

namespace Cody.Core.Agent.Protocol
{
  public class ChatExportResult
  {
    [JsonProperty(PropertyName = "chatID")]
    public string ChatID { get; set; }
    [JsonProperty(PropertyName = "transcript")]
    public SerializedChatTranscript Transcript { get; set; }
  }
}
